server/routes/contact.js

```diff
@@ -29,7 +29,12 @@ router.post('/email', async (req, res) => {
       message: req.body.message,
     });
 
-    const mailResponse = await sendMail(contact.email, 'stacy.sealky.lee@gmail.com', contact.subject, contact.message);
+    const mailResponse = await sendMail(
+      contact.email,
+      'stacy.sealky.lee@gmail.com',
```

Comment:
hold on, I forgot why my email is hard coded here. why?

Reply:
I don't know, that has only changed because of eslint formatting